hello_world.py

```diff
@@ -1 +1,34 @@
-print("hello world")
+# importing random to make use of it for tuples
+import random
+
+print("Hello World")
+
+# Closes Issue: #9
+# Asks the user to enter their name and displays it to them
+userName = input("Enter your name: ")
+print("Hello %s!" % userName)
+
+# Closes Issue: #8
+# Picks a random fruit from the tuple and tells the user to eat it
+fruits = ("apple", "banana", "cherry", "strawberry", "pineapple", "mango",
+          "grape", "orange")
+print("You should eat a %s." % fruits[random.randint(0, len(fruits))])
```

Comment:
Nicee 👍

Just for the reference we can also do something like `random.choice(fruits)` xD